tests/FSharpLint.Core.Tests/Rules/Conventions/AvoidTooShortNames.fs

```diff
@@ -162,3 +162,15 @@ async {
 } |> Async.RunSynchronously |> ignore<int>"""
 
         Assert.IsTrue this.NoErrorsExist
+
+    [<Test>]
+    member this.AvoidTooShortNamesShouldProduceError_15() =
+        this.Parse """
+type SomeType =
+    | SomeConstructor of int * string * bool
```

Comment:
@Mersho that's not a constructor; SomeType is a DU (maybe let's rename it to SomeDU, and rename SomeConstructor to SomeMember)